src/core/issues-notes.js

```diff
@@ -22,7 +22,18 @@ export function run(conf, doc, cb) {
       $issueSummary = $(
         "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
       ),
-      $issueList = $issueSummary.find("ul");
+      $issueList = $issueSummary.find("ul"),
+      closedIssues = [];
+    closedIssues = $ins.filter(function (index, item) {
```

Comment:
I think here we actually want something like this:

```JS
    const closedIssues = Array.from($ins)
       // only issues with a number that are not in ghIssues
      .filter(({ dataset: { number } }) => number && !ghIssues[dataNum])
      .forEach(elem => {
        const msg = `Github issue ${dataNum} was closed on GitHub, so removing from spec`;
        pub("warn", msg);
        elem.remove(); // remove from DOM
      });
```

As elements above get removed from the DOM tree, you then would need to also filter them out of `$ins` on line 37 (otherwise, we are doing needless work on things that are not in the DOM tree). You can filter DOM nodes that are not longer attached using `document.contains(element)`.

Reply:
Fixed typo above... didn't need `const closedIssues = ` at all.

Reply:
Thanks marcos :smiley:

Reply:
>You can filter DOM nodes that are not longer attached using `document.contains(element)`.

(Checking `element.parentNode` will be better for performance.)

Reply:
I'll look into this too

Reply:
@saschanaz is right. Though generally we prefer readability/maintainability over performance (unless it really needs super high performance code). I'm happy with either, so long as it's clear what the code is doing and why.

Reply:
Ok, I will be updating my work here very soon :smile:

Reply:
The reason you have used `Array.from` is to avoid another declaration of an array right ? So basically you wanted a use and throw sort of a method. Cool got it :+1: